src/pages/designing/index.mdx

```diff
@@ -59,20 +66,58 @@ IBMers can learn more about accessing design software in the
+| ------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [_Brand Center_](https://www.ibm.com/brand/)                        | IBM Brand Center is the home base for the IBM Brand story, visual brand elements, guidelines, and assets.                                                                                                                                                                            |
+| [_Brand Systems_](https://www.ibm.com/brand/experience-guides)      | The IBM brand systems have been developed for various IBM businesses, audiences, categories, and offerings. Read them to understand the rationale behind every visual and verbal detail.                                                                                             |
+| [_IBM Design Language_](https://www.ibm.com/design/language/)       | The IBM Design Language provides the guidance and assets used to express the IBM brand. You’ll fully understand what drives IBM’s design philosophy and principles, and be in a position to make informed choices for your product work.                                             |
+| [_Carbon Design System_](http://carbondesignsystem.com/)            | Carbon is IBM’s open source design system for products and digital experiences. The system is the visual expression of the IBM Design Language and consists of working code, design tools and resources, and human interface guidelines.                                             |
+| [_IBM Style_](https://ibmdocs-test.mybluemix.net/docs/en/ibm-style) | IBM Style is the corporate reference for all IBM writers and editors and applies to all content published by IBM. Its purpose is to ensure that content for external audiences is grammatically correct, clear, consistent, appropriate for global audiences, and easy to translate. |
+
+## Designing with Carbon for IBM.com
+
+### Component usage guidance
+
+You’ll learn a lot about designing with Carbon for IBM.com by reading through the component documentation. Most components include anatomy images, details about variations and modifiers where available, and content guidance that provides character count limits for component areas. You can also learn by looking at the components in Storybook.
+
+We are always working to make our guidance better and if there is information that you cannot find, please reach out to us on [Slack](https://ibm-studios.slack.com/archives/C2PLX8GQ6).
```

Comment:
Would be great to spell out the channel name: #carbon-for-ibm-dotcom

`Maybe "please reach out to us on Slack at [#carbon-for-ibm-dotcom](https://ibm-studios.slack.com/archives/C2PLX8GQ6) channel`